src/graph/InstallPluginExecutor.cpp

```diff
@@ -0,0 +1,78 @@
+/* Copyright (c) 2020 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "base/Base.h"
+#include "graph/InstallPluginExecutor.h"
+#include <dlfcn.h>
+
+namespace nebula {
+namespace graph {
+
+InstallPluginExecutor::InstallPluginExecutor(Sentence *sentence,
+                                             ExecutionContext *ectx)
+    : Executor(ectx, "install_plugin") {
+    sentence_ = static_cast<InstallPluginSentence*>(sentence);
+}
+
+Status InstallPluginExecutor::prepare() {
+    // Move this part of the operation to meta
```

Comment:
Why we `Move this part of the operation to meta`, If this PR is later for ldap, I suppose the graphd is the one to load?

Reply:
please ignore the comments, i will delete.

The current implementation logic is:
The install plugin just goes to the meta registration information,
uninstall plugin go to metad to delete registration information

The real place to open the so file is when in use, close the so file after use
For example,  ldap
graphd goes to metad to get the plugin information, then open the so file, and close the so file after use.
so here open so file and use so in graphd